templates/navigation.hbs

```diff
@@ -1,11 +1,11 @@
     <div class="nav-container">
       <div class="container">
-        <div class="pure-menu pure-menu-horizontal">
+        <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
```

Comment:
Not sure it's very useful in here...

Reply:
This ARIA landmark is required because main navigation isn't wrapped in a `<nav>` element. If we were to convert to HTML5 tags with semantic meaning, this role would be optional. However, as it is, screen reader users are likely having a difficult time locating the page's primary navigation.

You can read more on the topic from [Deque University's axe-core specification](https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html). Scanning the site without this landmark results in accessibility errors, so while you are unsure if this is useful, I've ran several audits of docs.rs and I respectfully submit that you're incorrect.

Reply:
> I respectfully submit that you're incorrect.

This part was too much (and is everything except respectful). I see your point and your arguments make sense so I now think that you're right but no need to make your point this bold, it makes conversation a bit less comfortable...